lib/fastlane/actions/verify_build.rb

```diff
@@ -67,7 +67,9 @@ def self.update_with_profile_info(app_path, values)
         values['provisioning_uuid'] = plist['UUID']
         values['team_name'] = plist['TeamName']
 
-        raise "Inconsistent identifier found" unless plist['Entitlements']['application-identifier'] == "#{values['team_identifier']}.#{values['bundle_identifier']}"
+        split_bundle_identifier = plist['Entitlements']['application-identifier'].partition('.').last
```

Comment:
is there a reason to use `partition` here instead of `split`? no problem with it I don't think, just curious!